pyp5js/pyp5js.py

```diff
@@ -557,7 +560,10 @@ def copy(*args):
     return _P5_INSTANCE.copy(*args)
 
 def filter(*args):
-    return _P5_INSTANCE.filter(*args)
+    if (len(args) > 1) and (type(args[1]) == type([])):
```

Comment:
It's safer to check if the first argument is a callable one instead of this type checking. For exemple,  `args[1]` can be a `set` or a `tuple` and thus this code will fail. So you can change this line to:

```python
if (len(args) > 1) and callable(args[0]):
```